File: triatlon-cs/Model/Result.cs
using System;

namespace Model
{
    [Serializable]
    public class Result
    {
    
        // Properties

        private int _id;
        private string _name;
        private int _points;
        private string _reversedName;

        public int Id
        {
            get => _id;
            set => _id = value;
        }
    
        public string Name
        {
            get => _name;
            set => _name = value;
        }
    
        public int Points
        {
            get => _points;
            set => _points = value;
        }
    
        public string ReversedName
        {
            get => _reversedName;
            set => _reversedName = value;
        }
    
        // Lifecycle

        public Result(int id, string name, string reversedName, int points)
        {
            _id = id;
            _points = points;
            _reversedName = reversedName;
            _name = name;
        }

    }
}

File: triatlon-java/model/src/main/java/triatlon/model/Result.java
package triatlon.model;

// AthleteResult

import java.io.Serializable;

public class Result implements Serializable {

    // Private Properties

    private Integer id;
    private String name;
    private Integer points;
    private final String reversedName;

    // Lifecycle

    public Result(Integer id, String name, String reversedName, Integer points) {
        this.id = id;
        this.name = name;
        this.reversedName = reversedName;
        this.points = points;
    }

    // Getters & Setters

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public String getNameReversed() {
        return this.reversedName;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getPoints() {
        return points;
    }

    public void setPoints(Integer points) {
        this.points = points;
    }

}
